content/docs/command-reference/remove.md

```diff
@@ -1,25 +1,25 @@
 # remove
 
-Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
+Remove stage entry, remove `.gitignore` entry and unprotect outputs.
```

Comment:
```suggestion
Remove stage, `.gitignore` entry, and unprotect outputs.
```

Reply:
I thought message should be identical with one in the core repo https://github.com/iterative/dvc/blob/81a4cea71a5a5d54d68575d8c4cacba7ad5563f3/dvc/command/remove.py#L24

I think your suggestion is beitter but probably it should be applied to the core repo first.

Reply:
> I thought message should be identical with one in the core repo

Not here 🙂 just in the code blocks such as under **Synopsis** or command output in examples. No need to apply this exactly to core. Thanks